src/generic/UnivPoly.jl

```diff
@@ -30,23 +30,21 @@ elem_type(::Type{UniversalPolyRing{T}}) where {T<:RingElement} = UnivPoly{T}
 
 parent_type(::Type{UnivPoly{T}}) where {T<:RingElement} = UniversalPolyRing{T}
 
-function mpoly_ring(S::UniversalPolyRing{T}) where {T<:RingElement}
-  return S.mpoly_ring::mpoly_ring_type(T)
-end
+mpoly_ring(S::UniversalPolyRing) = base_ring(S)
```

Comment:
I think this can even be removed entirely without breaking anything as it isn't exported.

Reply:
Since this is a breaking change anyway, I'd be fine with removing it -- as long as the change log explicitly documents this and what one should do instead.

Reply:
(well and since we are not really using it -- but of course there could be 3rd party code using it...)